contrib/debian/rules

```diff
@@ -1,24 +0,0 @@
-#!/usr/bin/make -f
```

Comment:
This is a [required file](https://www.debian.org/doc/manuals/maint-guide/dreq.en.html#rules).